src/renderers/shared/reconciler/__tests__/ReactUpdates-test.js

```diff
@@ -433,10 +433,15 @@ describe('ReactUpdates', function() {
     root = ReactTestUtils.renderIntoDocument(root);
 
     function expectUpdates(desiredWillUpdates, desiredDidUpdates) {
-      expect(willUpdates).toEqual(desiredWillUpdates);
```

Comment:
You could probably just make this `expect(willUpdates.slice()).toEqual(desiredWillUpdates)`. That will create a new array so that emptying out the original should be ok.